.github/renovate.json

```diff
@@ -9,7 +9,8 @@
         "^\\.github\\/workflows\\/.*\\.yml"
       ],
       "matchStrings":[
-        "terraform_version: (?<currentValue>.*?)  # renovate: datasource=(?<datasource>.*?) depName=(?<depName>.*?)\n"
+        "terraform_version: (?<currentValue>.*?)  # renovate: datasource=(?<datasource>.*?) depName=(?<depName>.*?)\n",
+        "terraform-version: (?<currentValue>.*?)  # renovate: datasource=(?<datasource>.*?) depName=(?<depName>.*?)\n"
```

Comment:
If I read `.github/workflows/check-terraform-syntax.yml` correctly, only the second version is needed, this isn't used anywhere else. Another option would be to change the variable name in that workflow to `terraform_version` in order to be more consistent.